dev/src/write-batch.ts

```diff
@@ -575,12 +575,33 @@ export class WriteBatch implements firestore.WriteBatch {
       api.BatchWriteResponse
     >('batchWrite', request, tag);
 
-    return (response.writeResults || []).map((result, i) => {
+    const writeResults = response.writeResults || [];
```

Comment:
Do you mind removing `GCF_IDLE_TIMEOUT_MS` (the unrelated lint complaint above)?

Reply:
it's being used further down in `_shouldCreateTransaction()`.